tests/test-cases/Group2-Imagec/2-10-Verify-Logfile.robot

```diff
@@ -3,8 +3,8 @@ Resource  ../../resources/Util.robot
 
 *** Test Cases ***
 Test
-    ${result}=  Run Process  ${bin-dir}/imagec -standalone -reference photon -logfile foo.log  shell=True  cwd=/
+    ${result}=  Run Process  ${bin-dir}/imagec -standalone -reference photon -logfile foo.log  shell=True  cwd=/tmp
```

Comment:
imagec has a destination parameter so I suggest using that instead of doing this